Projects/Server/main.cpp

```diff
@@ -8,19 +8,35 @@
 
 int main()
 {
+    size_t p0win{ 0 };
+    size_t p1win{ 0 };
+    size_t draw{ 0 };
+    size_t gameSample{ 500 };
+
     GameHandler gameHandler{};
 
     //AbstractPlayer::Ptr player0 = std::make_shared<StackAI::StackPlayer>();
-    //AbstractPlayer::Ptr player1 = std::make_shared<TranqAI::TranqPlayer>();
+    AbstractPlayer::Ptr player1 = std::make_shared<TranqAI::TranqPlayer>();
 
     //AbstractPlayer::Ptr player0 = std::make_shared<RandomPlayer>();
     AbstractPlayer::Ptr player0 = std::make_shared<InteractivePlayer>();
-    AbstractPlayer::Ptr player1 = std::make_shared<RandomPlayer>();
-
-    gameHandler.init(player0, player1);
-
-    //for (size_t repeat = 0; repeat < 10; ++repeat)
-        gameHandler.play();
-
+    //AbstractPlayer::Ptr player1 = std::make_shared<RandomPlayer>();
+    for (size_t i{ 0 }; i < gameSample; ++i) {
```

Comment:
I like the idea of adding repetition on tracking support, but let's try to keep main.cpp absolutely to the minimum, only containinh hyperparameters. Can you factor the support for repetition+tracking into its own class?

Reply:
Implemented the class GameCounter which keeps track of the stats associated with repetitive game play such as wins, draw, games played. Also, handles the game repetition.